Integrations/integration-AWS_Sagemaker.yml

```diff
@@ -0,0 +1,101 @@
+commonfields:
+  id: AWS Sagemaker
+  version: -1
+name: AWS Sagemaker
```

Comment:
This is very specific to Demisto AWS Sagemaker model, so I think it should be called `Demisto AWS Sagemaker phishing classifier `